examples/api/lib/widgets/overlay/overlay_portal.0.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for OverlayPortal
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Code Sample',
+      home: Scaffold(
+        appBar: AppBar(title: const Text('OverlayPortal Example')),
+        body: const Center(child: ClickableTooltipWidget()),
+      ),
+    );
+  }
+}
+
+class ClickableTooltipWidget extends StatefulWidget {
+  const ClickableTooltipWidget({super.key});
+
+  @override
+  State<StatefulWidget> createState() => ClickableTooltipWidgetState();
+}
+
+class ClickableTooltipWidgetState extends State<ClickableTooltipWidget> {
+  final OverlayPortalController _tooltipController = OverlayPortalController();
+
+  @override
+  Widget build(BuildContext context) {
+    return TextButton(
+      onPressed: _tooltipController.toggle,
+      child: DefaultTextStyle(
+        style: DefaultTextStyle.of(context).style.copyWith(fontSize: 50),
+        child: OverlayPortal(
+          controller: _tooltipController,
+          overlayChildBuilder: (BuildContext context) {
+            return const Positioned(
+              right: 50,
+              bottom: 50,
```

Comment:
Is this going to be positioned relative to the bottom right of the screen or the bottom right of the `Text` widget?  If the former, how could I position relative to the Text widget if I wanted to?  If the latter, how could I position relative to the entire screen?

Maybe consider another example or something so both of those situations are covered, if the answer isn't totally trivial.

Reply:
In overlay's coordinates. I cross-referenced CompositeTransformTarget/Follower in `Overlay`'s docs I think.

Reply:
Oh I didn't. Added to https://github.com/flutter/flutter/issues/107903